css/properties/offset-path.json

```
@@ -101,6 +101,57 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "path-support": {
+          "__compat": {
+            "description": "Supports the <code>path()</code> function as a value.",
```

A tiny, tiny thing and then we'll be ready to merge. These description fragments don't typically have a trailing period (there's a handful of counterexamples, which are now my to do list to fix)

```suggestion
            "description": "Supports the <code>path()</code> function as a value",
```